### PR TITLE
release: prepare changelog for v0.34.11

### PR DESCRIPTION
**Pull request by cmwaters**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+## v0.34.11
+
+*June 18, 2021*
+
+This release improves the robustness of statesync; tweaking channel priorities and timeouts and
+adding two new parameters to the state sync config.
+
+Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermint).
```
Comment (cmwaters) [inline CHANGELOG.md:10]: Do we really need this for every release - Can't we just add it once at the top of the changelog?

**Review by marbar3778 — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (marbar3778) [reply CHANGELOG.md:10]: yea, I agree here. I guess for now leave it.

**Comment by cmwaters** [comment]
FYI I bumped the default TM version in this PR as well

**Review by marbar3778 — COMMENTED** [review]
looks good, thanks for doing htis

**Pull request merged**